test/verifier/verifier_test.go

```diff
@@ -207,7 +219,7 @@ func TestVerifier(t *testing.T) {
 					// Offset points at the last newline, increment by 1 to skip it.
 					// Turn a -1 into a 0 if there are no newlines in the log.
 					lastOff := strings.LastIndex(p.VerifierLog, "\n") + 1
-					t.Logf("%s: %v", n, p.VerifierLog[lastOff:])
+					t.Logf("%s: %v (%d unverified insns)", n, p.VerifierLog[lastOff:], len(spec.Programs[n].Instructions))
```

Comment:
Out of curiosity, why call it "unverified instructions"? Is it to avoid a confusion with the number of instructions the verifier walked? I initially thought this was a report from the verifier of how many instructions state pruning avoided :sweat_smile:

Reply:
Yep, maybe something like 'pre-verified' might've been clearer in hindsight. 'raw' would've been misleading, since this is the number of logical instructions including double-wide.

Reply:
How about `%d insn-long program`? I'm worried even "pre-verified" might mislead some into thinking it's related to the verifier.